packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -2743,7 +2728,7 @@ test('refund to one of two loans', async t => {
   await m.assertLike({
     allocations: {
       Aeth: aeth.make(9n),
-      Fee: run.makeEmpty(),
+      Fee: undefined,
```

Comment:
Would it be better to drop `Fee: undefined`, rather than have it present? I think both mean the same thing to this test.

Reply:
I was wrong. In an *assertLike*, `Fee: undefined` verifies that it's absent.